server/game/gamesteps/shootout.js

```diff
@@ -34,6 +34,7 @@ class Shootout extends Phase {
             this.opposingPosse = new ShootoutPosse(this, this.opposingPlayer, false);
         }
 
+        this.winnerCasualtiesMod = 0;
```

Comment:
This changes in `shootout.js` are no longer needed.

Reply:
Bah, thought I got all that cleaned up.

Reply:
just a reminder that this is waiting only for the cleanup. sorry if you have that already covered :)